test/metabase/test/gen2.clj

```diff
@@ -0,0 +1,294 @@
+(ns metabase.test.gen2
+  (:require [clojure.spec.alpha :as s]
+            [clojure.test.check.generators :as gen]
+            [clojure.tools.reader.edn :as edn]
+            [java-time :as t]
+            [metabase.models
+             :refer
+             [Activity Card Collection Dashboard DashboardCard DashboardCardSeries Database Dimension Field
+              FieldValues Metric MetricImportantField NativeQuerySnippet PermissionsGroup PermissionsGroupMembership
+              Pulse PulseCard PulseChannel Table User]]
+            [metabase.test :as mt]
+            [metabase.util.date-2.parse :as u.date]
+            [reifyhealth.specmonstah.core :as rs]
+            [reifyhealth.specmonstah.spec-gen :as rsg]
+            [talltale.core :as tt]
+            [toucan.db :as tdb]))
+
+(def ^:private edn-definitions-dir "./test/metabase/test/data/dataset_definitions/")
+
+(defonce ^:private sample-dataset
+  (edn/read-string {:eof nil, :readers {'t #'u.date/parse}}
+                   (slurp (str edn-definitions-dir "sample-dataset.edn"))))
+
+(defn- random-desc
+  "Return a random name from sample-dataset"
+  []
+  (second (rand-nth (nth (first sample-dataset) 2))))
```

Comment:
Why don't you just use something like https://github.com/paraseba/faker to generate the fake data? I mentioned it in the Google Doc and the meetings, but this is the same library we used to generate the fake descriptions of things that are in the sample dataset. It seems like generate new fake descriptions makes more sense than randomly picking a fake one we already created. 

Take a look at https://github.com/metabase/metabase/blob/722649aa20e449b3319a34b7260fd702fac27505/sample_dataset/metabase/sample_dataset/generate.clj to see the code we used

Reply:
Oh, I missunderstood.
When you mentioned the sample-data as a source for the descriptions, and when I grepped around I found that, so I assumed that was it, and that you had imported it once.

Same goes for talltale instead of faker

Reply:
Talltale might be better than faker -- looks like it has more recent activity. I don't think it makes a huge difference whether we use talltale or faker or both

Reply:
Ended up replicating the code that was used to generate the products. it wasn't even faker, but it makes more sense than faker or talltale (they don't have generators for categories/products).